### PR TITLE
chore(eslint): Fix some ESLint warnings

### PR DESCRIPTION
**Pull request by pdehaan**
r? @meandavejustice @kaganjd

### CODE REVIEW

File: data/controls.js [hunk 1]
```diff
@@ -4,34 +4,34 @@
  * http://mozilla.org/MPL/2.0/.
  */
 
-document.querySelector('.controls').style.display = 'none';
-document.querySelector('.video-wrapper').style.display = 'none';
+document.querySelector(".controls").style.display = "none";
```
Comment (meandavejustice) [inline data/controls.js]: why `"` over `'`?

Comment (pdehaan) [inline data/controls.js]: No reason at all. I think when I did the initial .eslintrc file, I tried both and we used double quotes 60% of the time and single quotes the other 40%.
I'm happy to switch everything to single quotes if you prefer. It should be a trivial fix with the ESLint `--fix` flag which will swap them all for us.

Comment (meandavejustice) [inline data/controls.js]: yeah let's do single quotes to be consistent with testpilot, otherwise I'll get caught up on it all the time :)

Comment (pdehaan) [inline data/controls.js]: Done!

**Comment by meandavejustice** [comment]
Thank you @pdehaan

**Comment by pdehaan** [comment]
We should probably do #52 (Travis-CI integration) soon, so we'll have a full time ESLint shame-bot for each commit.

**Pull request merged**